website/client/components/groups/publicGuildItem.vue

```diff
@@ -147,6 +147,9 @@ export default {
     isMember () {
       return this.isMemberOfGroup(this.user, this.guild);
     },
+    isTier (tier) {
+      return this.isTier(tier, this.guild);
```

Comment:
hmm, there's already an isTier method available from the mixin, so you can remove this and just call isTier from the code, otherwise don't you risk overwriting the isTier from the mixin?

Reply:
Maybe? I'll try that.